Packs/Slack/Integrations/Slack/Slack.py

```diff
@@ -1311,8 +1311,20 @@ def slack_send():
     thread_id = demisto.args().get('threadID', '')
     severity = demisto.args().get('severity')  # From server
     blocks = demisto.args().get('blocks')
+    entry_object = demisto.args().get('entryObject')  # From server
     entitlement = ''
 
+    if message_type == MIRROR_TYPE:
+        tags = argToList(demisto.params().get('filtered_tags', []))
+        entry_tags = entry_object.get('tags', [])
+
+        if tags and not entry_tags:
```

Comment:
`entry_tags` will never be there in versions under 6.1. We need to explain it properly in the parameter description